### PR TITLE
Passed -> Matched, for clarity

### PR DESCRIPTION
**Pull request by fabianvf**
Signed-off-by: Fabian von Feilitzsch <fabian@fabianism.us>

### CODE REVIEW

**Comment by fabianvf** [comment]
I think tests may be failing at least in part because the tests are wrong, going to double check

File: demo-output.yaml [hunk 1]
```diff
@@ -149,6 +144,35 @@
       file: file:///analyzer-lsp/examples/golang/main.go
       lineNumber: 9
   extras: []
+- ruleID: lang-ref-001
```
Comment (fabianvf) [inline demo-output.yaml:147]: I think this should have been there all along

File: demo-output.yaml [hunk 2]
```diff
@@ -27,11 +27,6 @@
       innerText: "\n      io.fabric8\n      kubernetes-client-api\n      6.0.0\n    "
       matchingXML: <groupId>io.fabric8</groupId><artifactId>kubernetes-client-api</artifactId><version>6.0.0</version>
   extras: []
-- ruleID: chain-pom-002
```
Comment (fabianvf) [inline demo-output.yaml:30]: I don't think anything I did should lead to this being excluded?

Comment (shawn-hurley) [reply demo-output.yaml:30]: with no incidents, I wonder if we are returning the wrong value in the chain?

**Comment by shawn-hurley** [comment]
This looks good to me thank you this is going to make everything much clear

File: demo-output.yaml (hunk 2 above)
Comment (shawn-hurley) [reply demo-output.yaml:30]: or if it really should not be matched

**Comment by fabianvf** [comment]
@shawn-hurley is that approval to merge?

**Comment by shawn-hurley** [comment]
I think so yeah

File: provider/builtin/provider.go [hunk 3]
```diff
@@ -151,14 +151,17 @@ func (p *builtinProvider) Evaluate(cap string, conditionInfo []byte) (lib.Provid
...
 		query := cond.XML.XPath
+		fmt.Printf("\n\n\n Search Pattern: %v", cond.XML)
 		if query == "" {
 			return response, fmt.Errorf("Could not parse provided xpath query as string: %v", conditionInfo)
 		}
 		//TODO(fabianvf): how should we scope the files searched here?
 		var xmlFiles []string
 		if len(cond.XML.Filepaths) == 0 {
 			pattern := "*.xml"
+			fmt.Printf("\n\n\n Search Pattern: %v", xmlFiles)
```
Comment (shawn-hurley) [inline provider/builtin/provider.go:162]: 👍🏻 way to remove debugging!

**Pull request merged**